.github/workflows/lint-markdown.yml

```diff
@@ -0,0 +1,23 @@
+name: Lint
+
+on:
```

Comment:
- Add .mdx linter (https://github.com/mdx-js/eslint-mdx)
- Only trigger when there are changes to `.md/.mdx` file